packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -363,9 +365,9 @@ export const INVITE_PSM_COMMITTEE_MANIFEST = harden(
   }),
 );
 
-/** @type {import('@agoric/vats/src/core/manifest.js').BootstrapManifest} */
+/** @type {BootstrapManifest} */
 export const PSM_MANIFEST = harden({
-  /** @type {import('@agoric/vats/src/core/manifest.js').BootstrapManifestPermit} */
+  /** @type {import('@agoric/vats/src/core/lib-boot.js').BootstrapManifestPermit} */
```

Comment:
implied by being a property of BootstrapManifest
```suggestion
```

Reply:
looks like the `BootstrapManifest` declaration has to go inside `harden()` too.